File: Functions/Notes.py
# Sasha Rinitra, Functions Notes Python

# Function is an action stored in a key word

def add():
    numOne = int(input("Give me a number:\n"))
    numTwo = int(input("Give me a number:\n"))
    print(numOne+numTwo)

add()

#parameters go in parenthesis separated by commas
number = int(input("Can I get a number:\n"))
def add(numOne, numTwo):
    print(numOne+numTwo)

add(12,20) #arguments are given when the function is called AND must match the number of parameters
add(int(input("Give me a number:\n")), number)



def user(word):
    return input(f"Tell me a {word}:\n")
     

answer1 = user("name")
answer2 = user("verb")
answer3 = user("place")
print(f"{answer1} was {answer2} and somehow got to {answer3}.")

File: Functions/Notes.c
// Sasha Rinitra, Functions Notes C
#include <stdio.h>
// void add(){
//  }
//int add(void){
    //printf("%d\n", 6+8); 
    //return 6+8;
//}
//parameters
//void add(int num1, int num2){
  // printf("%d\n", num1+num2)
//}
// continued?
char input(char type[20]){
    char answer[50];
    printf("Please give me a %s:\n", type);
    scanf("%s", answer);
    return answer;
}
int main(void) {
  //printf("Hello World\n");
  //add();
  //add(3+5);
  printf("%s was %s until they somehow reached %s". input("name"), input("verb"), input("place"));
  return 0;
}